examples/receive.py

```diff
@@ -31,11 +31,13 @@ def main():
     if len(sys.argv) >= 2:
         rfxcom_device = sys.argv[1]
     else:
-        rfxcom_device = '/dev/serial/by-id/usb-RFXCOM_RFXtrx433_A1Y0NJGR-if00-port0'
+        rfxcom_device = ('192.168.2.6', 10001)
```

Comment:
Please revert the example code

Reply:
Right. Was not my intention to change defaults ofcourse. 

But to document a network example, should i duplicate the files or shall i add this comment in there?
```# For network device use:   rfxcom_device=('1.2.3.4', 10001), transport_protocol=RFXtrx.PyNetworkTransport```